src/components/styled/StyledPagination.tsx

```diff
@@ -0,0 +1,37 @@
+import { Pagination, PaginationProps } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import theme from "theme";
+
+const grey100 = theme.colors.grey100;
+
+const StyledPagination = styled(Pagination)<PaginationProps>({
+  color: grey100,
+
+  "& .MuiPaginationItem-root": {
+    color: grey100,
+    borderRadius: "0px",
+    border: `1px solid ${grey100}`,
+    margin: "0px",
+  },
+  "& .MuiPaginationItem-previousNext > svg": {
+    color: grey100,
+  },
+  "& .MuiPaginationItem-previousNext": {
+    borderRadius: "0px 6px 6px 0px",
+  },
+  "& li:first-child > button": {
+    borderRadius: "6px 0px 0px 6px",
+  },
+
+  "& .MuiPaginationItem-ellipsis": {
+    paddingTop: "5px !important",
+    paddingBottom: "5px !important",
+  },
+
+  "& .Mui-selected": {
+    color: "white",
+    background: theme.colors.grey400,
```

Comment:
In this case you are only changing the background color. It is good practice to use `background-color` in this case because using `background` property here will reset the rest of the properties background shorthand uses like repeat, position, color etc.  Unless you meant to reset the rest of the properties to their default, then you can keep it as is

Reply:
I did mean to reset all the other properties 👍